tests/docs/test_docs.py

```diff
@@ -11,16 +11,22 @@
 from ..conftest import _CHEM_INSTALLED
 from .utils import extract_code_blocks
 
-doc_files = ["README.md", *Path("docs/userguide/").rglob("*.md")]
+doc_files = [Path("README.md"), *Path("docs/userguide/").rglob("*.md")]
 """Files whose code blocks are to be checked."""
+
 doc_files_pseudocode = [Path("docs/userguide/campaigns.md")]
-"""Files which contain pseudocode that needs to be checked using fixtures."""
+"""Files containing pseudocode that needs to be checked with injected fixtures."""
+
+
+def paths_to_params(*paths: Path):
```

Comment:
Could you explain why this change was necessary?

Reply:
It was just really annoying to fix broken tests because the only info you get is like "test fails for example_27", because the parametrization just passes the example files coming from `rglob`. The change adds some nice IDs such that you know exactly which example failed and you can even nicely select the respective test to run in your IDE:

<img width="355" alt="image" src="https://github.com/emdgroup/baybe/assets/23265127/fb88f0b6-3967-4910-a1c8-834563e61cd7">